data/172/939/604/1/1729396041.geojson

```diff
@@ -36,11 +41,19 @@
     ],
     "wof:country":"US",
     "wof:geomhash":"7e8336d5af580d3af66a807f2f24ad62",
-    "wof:hierarchy":[],
+    "wof:hierarchy":[
+        {
+            "continent_id":102191575,
+            "country_id":85633793,
+            "county_id":102084305,
+            "localadmin_id":1729396041,
+            "region_id":85688563
+        }
+    ],
     "wof:id":1729396041,
-    "wof:lastmodified":1605055260,
+    "wof:lastmodified":1613521985,
     "wof:name":"27",
-    "wof:parent_id":-1,
+    "wof:parent_id":102084305,
```

Comment:
For the new `localadmin` we're importing like this one... often times they shouldn't be part of a label hierarchy. But this one is marked true for that... and it's more like a statistical gore than not?

Reply:
Here's the record after all commits on this branch: https://raw.githubusercontent.com/whosonfirst-data/whosonfirst-data-admin-us/stepps00/locality-localadmin/data/172/939/604/1/1729396041.geojson

This record _is_ marked with `"mz:hierarchy_label":1` and does not have a `wof:statistical_gore` property set.

There were 101 new `localadmin` records added to this repo (this record being one of them) - mostly to handle new/removed features from the US Census file. Are we proposing to update properties on all `localadmin`?

Reply:
I see, let's ignore this for now. 

In Nebraska there are a few of these oddballs that the US Census seems to add in to create a continuous fabric but are otherwise a bit odd. See https://spelunker.whosonfirst.org/id/85688563/descendants/?exclude=nullisland&placetype=localadmin.